res/map/Asphalt_Maze.map

```diff
@@ -0,0 +1,11 @@
+  FR  FR  FR  FR  FR  GR  GR  GR  GR  GR  GR  GR  GR 1FC  GR
+  FR  FR  FR  GR  GR  RD  RD  RD  RD  RD  RD  RD 1FC 1HQ 1FC
+  FR  FR  FR  GR  FR  RD  FR  RD  FR  RD  FR  RD  GR 1FC  GR
+  FR  GR  GR  RD  RD  RD  RD  CT  RD  RD  RD  RD  RD  RD  GR
+  FR  GR  FR  RD  FR  RD  FR  RD  FR  RD  FR  RD  FR  RD  GR
+  GR  RD  RD  RD  RD  CT  RD  RD  RD  CT  RD  RD  RD  RD  GR
+  GR  RD  FR  RD  FR  RD  FR  RD  FR  RD  FR  RD  FR  GR  FR
+  GR  RD  RD  RD  RD  RD  RD  CT  RD  RD  RD  RD  GR  GR  FR
+  GR 0FC  FR  RD  FR  RD  FR  RD  FR  RD  FR  GR  FR  FR  FR
+ 0FC 0HQ 0FC  RD  RD  RD  RD  RD  RD  RD  GR  GR  FR  FR  FR
+  GR 0FC  GR  GR  GR  GR  GR  GR  GR  GR  FR  FR  FR  FR  FR
```

Comment:
This file looks correct, but in-game, the map is missing the last row.
![asphaltmaze](https://user-images.githubusercontent.com/16439262/34311277-44fd0ca0-e722-11e7-901c-f6e3e0b22656.png)